src/praisonai/praisonai/cli/interactive/async_tui.py

```
@@ -839,7 +843,41 @@ def _handle_command(self, command: str) -> bool:
-        
+
+        elif cmd in ("code-review", "security-review"):
+            # Review the uncommitted (or staged/per-file) diff using the shipped
+            # read-only "review" agent preset. `security` swaps in a security
+            # rubric; both attach the fenced diff so the reviewer sees the exact
+            # changed lines. No writes are possible during review.
+            security = cmd == "security-review"
+            staged = False
+            file_path = None
+            for token in args.split():
+                if token == "--staged":
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "token"

(S105)